modules.d/99base/dracut-lib.sh

```diff
@@ -844,11 +836,11 @@ add_mount_point() {
 wait_for_mount() {
     local _name
     _name="$(str_replace "$1" '/' '\\x2f')"
-    printf '. /lib/dracut-lib.sh\nismounted "%s"\n' $1 \
+    printf '. /lib/dracut-lib.sh\nismounted "%s"\n' "$1" \
         >> "$hookdir/initqueue/finished/ismounted-${_name}.sh"
     {
-        printf 'ismounted "%s" || ' $1
-        printf 'warn "\"%s\" is not mounted"\n' $1
+        printf 'ismounted "%s" || ' "$1"
```

Comment:
Should not this be

`
printf ' "\"%s\" is mounted || ' "$1"
`
or
`
printf ' "\"%s\" already mounted || ' "$1"
`
to be inline with the following
`
printf 'warn "\"%s\" is not mounted"\n' "$1"
`
?

Reply:
no.. inside `'` we don't have to quote `"`